src/components/PageNotFound/PageNotFound.js

```diff
@@ -1,23 +1,23 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import './style.css';
+import './style.scss';
 
 const PageNotFound = ({ path }) => {
   return (
-    <>
-      <div className="pageNotFound__opacity"></div>
-      <div className="pageNotFound__info">
-        <div className="pageNotFound__text">
+    <div className="PageNotFound">
+      <div className="PageNotFound__opacity"></div>
+      <div className="PageNotFound__info">
+        <div className="PageNotFound__text">
           <h2>Looks like you got lost</h2>
```

Comment:
Since we decided to use SCSS. Rather use the html tag to style it, could use className for h2 tag. e.g "PageNotFound__title". Same for h3 tag below. It is good practice to use classNames to add styles in css/sass.